.erda/migrations/pipeline/20220322-pipeline-archive-add-index.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `pipeline_archives` ADD INDEX `idx_status_timecreated` (`status`, `time_created`);
```

Comment:
time_created or time_end
you can add index for both fields.

Reply:
added